src/targets/mobile/main.jsx

```diff
@@ -69,6 +70,7 @@ const renderAppWithPersistedState = persistedState => {
   document.addEventListener('deviceready', () => {
     pingOnceADayWithState()
     store.dispatch(backupImages())
+    store.dispatch(incrementBootCount())
```

Comment:
If you need to access the global state only to persist the value with such kind of instruction, ,maybe you can add that behavior in the stateful component. Or in a HOC responsible to persist the state of a component:

```jsx
const withPersistency = WrappedComponent => class extends React.Component {
  constructor (props) {
    super(props)
    this.state = undefined
  }

  componentWillMount () {
    loadStateFormLocalStorage().then(state => this.setState(() => state))
  }

  componentWillUnmout () {
    saveStateToLocalStorage(this.state)
  }

  render () {
    <WrappedComponent {...this.state} />
  }
}
```

or something similar.